ratatool-sampling/src/test/scala/com/spotify/ratatool/samplers/BigSamplerAvroTest.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2016 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.ratatool.samplers
+
+import java.nio.ByteBuffer
+import java.util.Arrays
+
+import com.google.common.hash.Hashing
+import com.spotify.ratatool.Schemas
+import org.apache.avro.generic.GenericData.EnumSymbol
+import org.apache.avro.generic.{GenericData, GenericRecordBuilder}
+import org.scalatest.{FlatSpec, Matchers}
+
+class BigSamplerAvroTest extends FlatSpec with Matchers {
```

Comment:
We have some property based tests for hasher already defined in here https://github.com/spotify/ratatool/blob/master/ratatool-sampling/src/test/scala/com/spotify/ratatool/samplers/BigSamplerTest.scala#L36

I think it makes sense to consolidate those into the same file, and keep the JobTests in a separate file.

Reply:
Also another note outside scope of this PR, would be nice to rethink some of these as property based tests as well 😉 But not a blocker, up to you

Reply:
Fixed!

I agree it would be nice to rethink some of these tests! :-)